Maddori.Apple/Maddori.Apple/Global/UIComponent/View/EmptyReflectionView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  EmptyReflectionView.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/11/06.
+//
+
+import UIKit
+
+import SnapKit
+
+final class EmptyReflectionView: UIView {
+    
+    private enum Size {
+        static let calendarWidth: CGFloat = 60
+        static let calendarHeight: CGFloat = 65
+        static let calendarYOffset: CGFloat = -60
+    }
+    
+    // MARK: - property
+    
+    private let emptyReflectionImageView: UIImageView = {
+        let view = UIImageView()
+        view.image = ImageLiterals.imgEmptyCalendar
+        return view
+    }()
+    var emptyReflectionLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.emptyViewMyReflection
+        label.textColor = .gray700
+        label.font = .body3
+        label.textAlignment = .center
+        label.numberOfLines = 2
+        return label
+    }()
```

Comment:
p2;

```suggestion
    private let emptyReflectionLabel: UILabel = {
        let label = UILabel()
        label.text = TextLiteral.emptyViewMyReflection
        label.textColor = .gray700
        label.font = .body3
        label.textAlignment = .center
        label.numberOfLines = 2
        return label
    }()
```

외부에서 접근하지 않고 있고, let 으로 사용해도 될 것 같습니다 !

Reply:
[4c9ebe8](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/85/commits/4c9ebe84dc9227a5e7ea2780dfa0e172d8a27a6d)
감사합니다!!